src/test/java/camp/nextstep/edu/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,50 @@
+package camp.nextstep.edu.calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+/**
+ * Created by inbonk on 2019/11/25.
+ */
+class StringCalculatorTest {
+
+    @DisplayName("입력이 null 이면 결과는 0이다")
+    @Test
```

Comment:
```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    void nullInput(String source) {
```
ParameterizedTest를 적극적으로 활용해보는 것도 좋겠네요 ㅎ